dockers/docker-orchagent/base_image_files/monit_swss

```diff
@@ -39,5 +38,3 @@ check process buffermgrd matching "/usr/bin/buffermgrd -l"
 check process nbrmgrd matching "/usr/bin/nbrmgrd"
     if does not exist for 5 times within 5 cycles then alert
 
-check process vxlanmgrd matching "/usr/bin/vxlanmgrd"
-    if does not exist for 5 times within 5 cycles then alert
```

Comment:
Why remove monitoring for vxlanmgrd in 201811?

Reply:
I don't think vxlanmgrd exists on 201811

Reply:
You're correct. I was looking at the wrong branch :)